openfisca_web_api/wsgihelpers.py

```diff
@@ -125,3 +125,15 @@ def convert_date_to_json(obj):
         return obj.isoformat()
     else:
         return json.JSONEncoder().default(obj)
+
+
+def handle_error(error, ctx, headers):
+    raise respond_json(ctx,
+        dict(
+            error = dict(
+                code = 400,
+                errors = u"{}: {}".format(error.__class__.__name__, error.message),
```

Comment:
@fpagnoux I didn't catch it during the review of #104 but why is the key `errors = ` written with plural form, whereas the value is a string?

It would be better to override the `message` key. I'm doing a proposition in #111